### ISSUE TITLE
[Feature Request] Add a setting which when checked adds the Per-Domain rules for Google and YouTube sites

### ISSUE THREAD

**Issue by Victor239** [stoically/temporary-containers#511]
Google sites such as maps.google.com and youtube.com now use a consent webpage before allowing you to access their websites. With Automatic Mode enabled, you cannot access these websites unless you have the rules listed below ([supplied here](https://github.com/stoically/temporary-containers/issues/499#issuecomment-812493496)):

Domain Pattern:
`consent.youtube.com`

Exclude Target Domains:
`*.youtube.com`
`consent.google.*`
`googleads.g.doubleclick.net`

---

Domain Pattern:
`consent.google.*`

Exclude Target Domains:
`*.youtube.com`
`*.google.*`
`googleads.g.doubleclick.net`

---

This took me a while of testing and searching to figure out which I think would cause many to just give up and disable the extension entirely. Since these are extremely popular domains it would be helpful for new TC users to have an option to easily add the rules for these sites.

**Comment by CheariX** [comment]
Thank's! This was incredibly helpful.

**Comment by solna86** [comment]
I think this is a very important issue that is preventing a more widespread adoption of this addon, which I find exceptionally useful to make browsing more secure and private.

@Victor239 and @CheariX, there are better ways I think to solve this problem, but they are not fully satisfactory. One possibility is to block consent pop ups using Ublock Origin, which you might be using anyway to address privacy issues. Note this requires a custom Ublock rule.

I would be interested in hearing what @stoically thinks about this given that it is an issue that frustrates many users and drives them to plain containers, which are inferior, or no containers at all.

**Comment by Gitoffthelawn** [comment]
Just FYI, to my limited knowledge, those annoying consent pages are only given to specific IP ranges.

**Comment by solna86** [comment]
@Gitoffthelawn perhaps you are referring to captcha pages, unless I'm missing something?

I am referring to a Terms and Conditions pop up that Google, Google Maps, Youtube, etc show you on your "first" visit, and where you need to click accept. This might be an EU specific thing related to GPDR, though.

When using Temporary Containers, every visit looks like the first one to Google because there are no cookies.

I can block some of those popups with custom Ublock Origin rules, but there are still some glitches and annoyances left.

**Comment by Gitoffthelawn** [comment]
> @Gitoffthelawn perhaps you are referring to captcha pages, unless I'm missing something?
> 
> I am referring to a Terms and Conditions pop up that Google, Google Maps, Youtube, etc show you on your "first" visit, and where you need to click accept. This might be an EU specific thing related to GPDR, though.
> 
> When using Temporary Containers, every visit looks like the first one to Google because there are no cookies.
> 
> I can block some of those popups with custom Ublock Origin rules, but there are still some glitches and annoyances left.

Yes, Google only sends those popups to specific IP ranges.  Captcha's are a separate Google evil.
